docs/content/guides/dagster/recommended-project-structure.mdx

```diff
@@ -5,6 +5,10 @@ description: This guide talks about Dagster's recommendations on structuring lar
 
 # Structuring your Dagster project
 
+<Note>
+<strong>Got questions about our recommendations, or something to add?</strong> Join our [GitHub discussion]() to share how you organize your Dagster code.
```

Comment:
```suggestion
<strong>Got questions about our recommendations, or something to add?</strong> Join our [GitHub discussion](https://github.com/dagster-io/dagster/discussions/8972) to share how you organize your Dagster code.
```

🔧 Adding the link here - I think this is the right discussion (cc @yuhan)

Reply:
yes it is